tensorbay/dataset/dataset.py

```diff
@@ -78,6 +81,19 @@ def loads(cls: Type[_T], contents: Dict[str, Any]) -> _T:
         """
         return common_loads(cls, contents)
 
+    def keys(self) -> Tuple[str, ...]:
```

Comment:
will it work if `keys()` returns a `Gererator`

Reply:
It can work with returning a `Generator`, but should the `keys()` function return a `Generator`?

Reply:
Actually `keys` should return a `KeysView`

Reply:
https://docs.python.org/zh-cn/3/library/collections.abc.html#collections.abc.KeysView

Reply:
As a reference:
https://github.com/python/cpython/blob/master/Lib/_collections_abc.py#L782